code/game/sound.dm

```diff
@@ -26,12 +26,14 @@
 		if(get_dist(M, turf_source) <= maxdistance)
 			sound_or_datum(M, turf_source, source, input, vol, vary, frequency, falloff, channel, pressure_affected)
 
-/proc/sound_or_datum(mob/receiver, turf/turf_source, datum/source, input, vol as num, vary, frequency, falloff, channel = 0, pressure_affected = TRUE)
+/proc/sound_or_datum(mob/receiver, turf/turf_source, atom/source, input, vol as num, vary, frequency, falloff, channel = 0, pressure_affected = TRUE)
```

Comment:
im not sure what this change is supposed to accomplish since its a simple typecast and can as well be avoided with :

Reply:
if you look further back in the code this is the same atom/source that's in playsound(source of the sound, sound/datum)

the source of the sound has to be something that exists in the world, which a datum isnt, so im pretty sure i cant have the source be a datum